dev-tools/create_bwc_index.py

```diff
@@ -265,12 +265,21 @@ def generate_index(client, version, index_name):
   mappings['doc'] = {'properties' : {}}
   supports_dots_in_field_names = parse_version(version) >= parse_version("2.4.0")
   if supports_dots_in_field_names:
-    mappings["doc"]['properties'].update({
+
+    if parse_version(version) < parse_version("5.0.0-alpha1"):
+      mappings["doc"]['properties'].update({
         'field.with.dots': {
           'type': 'string',
           'boost': 4
         }
       })
+    else:
+      mappings["doc"]['properties'].update({
+        'field.with.dots': {
+          'type': 'text',
+          'index': True
```

Comment:
@bleskes You mean on a dynamically created field? I think the default is to create both a `text` and a `keyword` field. Or are you asking about `'index': True`?